### ISSUE TITLE
NTR: Anti-HLA-DQ  antibody positivity

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10615]
**Preferred term label:**
Anti-HLA-DQ  antibody positivity

**Synonyms**
Donor specific antibody positivity, 

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies (immunoglobulins) in the blood circulation that react against a human leukocyte antigen (HLA-DQ). DQ is one of several antigens involved in antibody-mediated rejection of organ transplants. 

**Parent term (use hpo.jax.org/app)**
HP:0030057: Autoimmune antibody positivity 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-0113-912X, 0000-0002-7371-8158

**Comment by GrieseAG** [comment]
Definition in publication: https://doi.org/10.1016/j.healun.2012.10.007

**Comment by pnrobinson** [comment]
@GrieseAG 
added as
Anti-HLA-DQ antibody positivity
HP:6001015
